azure/vars.tf

```diff
@@ -19,6 +19,12 @@ variable "vm_instances" {
   default     = 3
 }
 
+variable "use_scale_set" {
```

Comment:
Can we update the READMEs with the new vars?

Reply:
Updated.